docs/bin/plugin_formatter.py

```diff
@@ -58,6 +58,9 @@ def html_escape(text, quote=True):
 from ansible.utils.display import Display
 from ansible.utils._build_helpers import update_file_if_different
 
+sys.path.append('../../test/sanity/validate-modules')
```

Comment:
I'd prefer seeing this set up via env var, it'd be cleaner. And then, use a full import path. So maybe we could convince @mattclay to move these helpers to a better importable location?

Reply:
Please, this is a WIP. It makes no sense to review this now.

As I already mentioned this is a hack. This code will move into Ansible so it can be reused easier between test and documentation tools.